lifecycle-operator/chart/templates/keptnapp-crd.yaml

```
@@ -4,7 +4,9 @@ metadata:
   name: keptnapps.lifecycle.keptn.sh
   annotations:
     controller-gen.kubebuilder.io/version: v0.13.0
-    cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/keptn-certs'
+{{- if and .Values.global.caInjectionAnnotations (not (empty .Values.global.caInjectionAnnotations)) }}
```

Helm does the empty check for you, we can simplify the expression like the following: 
```suggestion
{{- if .Values.global.caInjectionAnnotations }}
```